docs/contributor/arch/006-telemetry-api-v1beta1.md

```diff
@@ -0,0 +1,33 @@
+# 6. Telemetry API v1beta1
```

Comment:
Number 5 seems to be missing from the dir. Is it omitted on purpose?

Reply:
Number 5 is not merged yet https://github.com/kyma-project/telemetry-manager/pull/758 🤣